<center><h1 style="line-height: 1.5;">
  Звіт Лабораторна робота №3 <br>
  ДОСЛІДЖЕННЯ БІПОЛЯРНИХ ТРАНЗИСТОРІВ
</h1></center>

<h4 style="line-height: 1.5;"> Мета роботи: вивчення принципу дії, основних властивостей та характеристик біполярного транзистора, ознайомлення з його основними параметрами і методикою їх вимірювання.</h4>

##### Варіант 2, тразистор 2N1613

Для дослідження вхідних характеристик транзистора **2N1613** зберемо схему наведену на малюнку 3.1. Як об'єкт управління виберемо джерело живлення VS1 (через пункти меню "Аналіз – Об'єкт управління"), встановлюємо наступні параметри: "Початкове значення" дорівнює "0", "Кінцеве значення" - 30, "Кількість випадків" - 4.

<center><img src="images/Scheme_1.png" width="400"/><h4>Рисунок 3.1 - Схема для дослідження</h4></center>

Вибираємо пункти меню "Аналіз – Аналіз постійного струму – Перехідні характеристики постійного струму..." Встановлюємо параметр "Початкове значення" рівним "0", параметр "Кінцеве значення" рівним 2 мА, а параметр "Кількість точок" рівним 1000. В результаті отримаємо вхідні характеристики біполярного транзистора (рис. 3.2).

<center><img src="images/Graph_1.png" width="450"/><h4>Рисунок 3.2 - Вхідні характеристики транзистора для схеми СЕ</h4></center>

Для дослідження вихідних характеристик біполярного транзистора у схемі **(СЕ)** зберемо схему, що наведена на малюнку 3.3. Як об'єкт управління виберемо джерело струму IS1. У меню "Контроль об'єкта виділення" встановлюємо "Початкове значення" рівним "0", "Кінцеве значення" – 2 мА, а "Кількість випадків" – 6.

<center><img src="images/Scheme_2.png" width="400"/><h4>Рисунок 3.3 - Схема для дослідження вихідних характеристик СЕ</h4></center>

Вибираємо пункти меню "Аналіз – Аналіз постійного струму – Перехідні характеристики постійного струму...". Встановлюємо "Початкове значення", що дорівнює "0", "Кінцеве значення" – 30 В, "Кількість точок" 1000. У якості вхідної змінної "Вхід" виберемо джерело VS1. В результаті отримаємо вихідні характеристики біполярного транзистора для схеми **(СЕ)** (рис. 3.4).

<center><img src="images/Graph_2.png" width="400"/><h4>Рисунок 3.4 - Вихідні характеристики для схеми СЕ</h4></center>

Для дослідження вхідних характеристик транзистора у схемі зі спільною базою **(СБ)** зберемо схему, що наведена на (рис. 3.5). В якості об’єкта управління виберемо джерело живлення VS1. У меню "Контроль об'єкта виділення" встановимо "Початкове значення" рівним "0", "Кінцеве значення" – 30, "Кількість випадків" – 4.

<center><img src="images/Scheme_3.png" width="400"/><h4>Рисунок 3.5 - Схема для дослідження вхідних характеристик СБ</h4></center>

Виберемо пункти меню "Аналіз – Аналіз постійного струму – Перехідні характеристики постійного струму...". Встановлюємо "Початкове значення", що дорівнює "0", "Кінцеве значення" – 100 мА, "Кількість точок" 1000. У якості вхідної змінної "Ввод" встановлюємо IS1. В результаті отримаємо вхідні характеристики біполярного транзистора для схем **(СБ)** (рис. 3.6).

<center><img src="images/Graph_3.png" width="400"/><h4>Рисунок 3.6 - Вхідні характеристики для схеми СБ</h4></center>

Для дослідження вихідних характеристик біполярного транзистора у схемі **(СБ)** зберемо схему, що наведена на (рис 3.7). Як об'єкт управління виберемо джерело струму S1. Встановимо "Початкове значення" рівним "0", "Кінцеве значення" – 100 мА, "Кількість випадків" – 6.

<center><img src="images/Scheme_4.png" width="400"/><h4>Рисунок 3.7 - Схема для дослідження вихідних характеристик СБ</h4></center>

Для побудови вихідних характеристик вибираємо "Аналіз - Аналіз постійного струму - Перехідні характеристики постійного струму...". Встановлюємо параметр "Початкове значення" дорівнює мінус 2 В, "Кінцеве значення" – 30 В, "Кількість точок" 1000. В якості вхідної змінної "Введення" виберемо джерело VS1. В результаті отримаємо вихідні характеристики біполярного транзистора для схеми **(СБ)** (рис. 3.8).

<center><img src="images/Graph_4.png" width="400"/><h4>Рисунок 3.8 - Вихідні характеристики для схеми СБ</h4></center>

За статистичними характеристиками для схем **(СЕ)** та **(СБ)** знайдемо h параметри для обох схем. При розрахунку параметрів використовуємо вихідну характеристику середнього та найближчого до середнього значення вхідного струму. Отримані значення параметрів h запишемо до таблиці 3.1.

Розрахунок значень h параметрів.
Вхідний опір:
$$
h_{11e} = \frac{\Delta U_{БЕ}}{\Delta I_Б}
$$
Коефіцієнт зворотного зв'язку по напрузі:
$$
h_{12e} = \frac{\Delta U_{БЕ}}{\Delta U_{КЕ}}
$$
Коефіцієнт передачі струму:
$$
h_{21e} = \frac{\Delta I_К}{\Delta I_{Б}}
$$
Вихідна провідність:
$$
h_{22e} = \frac{\Delta I_К}{\Delta U_{КЕ}}
$$

In [29]:
# Дані для СЕ та СБ у вигляді словників
data_se = {
    "ube": [772e-3, 822e-3, 828e-3],
    "uke": [1, 3, 10, 20],
    "ib":  [0.8e-3, 1.2e-3, 750e-6, 1.75e-3],
    "ik":  [73.65e-3, 76.31e-3, 53.58e-3]
}

data_sb = {
    "ube": [734e-3, 795e-3, 803e-3],
    "uke": [5, 25, 10, 0],
    "ib":  [40e-3, 60e-3, 25e-3, 85e-3],
    "ik":  [59.15e-3, 59.38e-3, 39.6e-3]
}

# Функція для розрахунку h-параметрів
def calc_h(data):
    ube, uke, ib, ik = data["ube"], data["uke"], data["ib"], data["ik"]
    h11 = (ube[1] - ube[0]) / (ib[3] - ib[2])
    h12 = (ube[2] - ube[1]) / (uke[3] - uke[2])
    h21 = (ik[1] - ik[2]) / (ib[1] - ib[0])
    h22 = (ik[1] - ik[0]) / (uke[1] - uke[0])
    return h11, h12, h21, h22

# Функція виводу
def show_result(name, hes, scale=1e3, unit="*10^-3"):
    print(f"=== Розрахункові параметри схеми ({name}) ===")
    print(f"h11e (вхідний опір)          = {hes[0]:.1f} Ом")
    print(f"h12e (зворотний зв'язок U)   = {hes[1]*scale:.2f} {unit}")
    print(f"h21e (β, підсилення по струму) = {hes[2]:.1f}")
    print(f"h22e (вихідна провідність)   = {hes[3]:.5f} См  →  {hes[3]*scale:.1f} мСм\n")
    
# Розрахунок
h_se = calc_h(data_se)
h_sb = calc_h(data_sb)
h_se_dv = (2200, 3.6e-4, 70, 0.0125e-3)

# Вивід результатів
show_result("СЕ", h_se, 1e3, "*10^-3")
show_result("СБ", h_sb, 1e4, "*10^-4")

=== Розрахункові параметри схеми (СЕ) ===
h11e (вхідний опір)          = 50.0 Ом
h12e (зворотний зв'язок U)   = 0.60 *10^-3
h21e (β, підсилення по струму) = 56.8
h22e (вихідна провідність)   = 0.00133 См  →  1.3 мСм

=== Розрахункові параметри схеми (СБ) ===
h11e (вхідний опір)          = 1.0 Ом
h12e (зворотний зв'язок U)   = -8.00 *10^-4
h21e (β, підсилення по струму) = 1.0
h22e (вихідна провідність)   = 0.00001 См  →  0.1 мСм



Таблиця вимірів та розрахунків 3.1
| Схема | $h_{11},\ Ом$|$h_{12}$|$h_{21}$|$h_{22},\ S(Сіменс)$|
|:--|:-----|:-----|:-----|:-----|
| СЕ розрахункове значення | 50 | $6 \cdot 10^{-4}$ | 56.8 | $1.33 \cdot 10^{-3}$ | 
| СБ розрахункове значення  | 1 | $8 \cdot 10^{-4}$ | 1 | $0.1 \cdot 10^{-3}$ |
| СЕ довідкове значення  | 2200 | $3.6 \cdot 10^{-4}$ | 70 | $0.0125 \cdot 10^{-3}$ |


Розрахуємо для схем СЕ та СБ значення фізичних параметрів $r_е$, $r_б$, $r_к$, $µ_{ек}$, а також коефіцієнтів α та β. Запишемо отримані дані у таблицю 3.2. 

$$
r_{e} = \frac{h_{12e}}{h_{22e}};
$$
$$
r_{б} = h_{11e} - (1 + h_{21e})\cdot \frac{h_{12e}}{h_{22e}};
$$
$$
r_x = \frac{1 + h_{21e}}{h_{22e}};
$$
$$
\mu_{вк} = \frac{h_{11e} \cdot h_{22e}}{(1 + h_{21e}) - h_{12e}};
$$
$$
\alpha = \frac{h_{21e}}{1 + h_{21e}};
$$
$$
\beta = h_{21e}.
$$

In [22]:
# Функція для розрахунку
def calc_r(data):
    h11, h12, h21, h22 = data[0], data[1], data[2], data[3]
    re = h12 / h22
    rb = h11 - (1 + h21) * re
    rx = (1 + h21) / h22
    mu = (h11 * h22) / ((1 + h21) - h12) 
    a = h21 / (1 + h21)          # коефіцієнт передачі струму α ≈ β/(β+1)
    b = h21
    return re, rb, rx, mu, a, b

def show_results(name, res, scale=1e3, unit="*10^-3"):
    print(f"=== Розрахункові параметри схеми ({name}) ===")
    print(f"re = {res[0]:.1f} Ом")
    print(f"rb = {res[1]:.1f} Ом")
    print(f"rx = {res[2]:.0f} Ом")
    print(f"μ  = {res[3]*scale:.5f} {unit}")
    print(f"α  = {res[4]:.4f}")
    print(f"β  = {res[5]:.1f}\n")

# Розрахунок
r_se = calc_r(h_se)
r_sb = calc_r(h_sb)
r_se_dv = calc_r(h_se_dv)
                                          
# Вивід
show_results("СЕ", r_se, 1e3, "*10^-3")
show_results("СБ", r_sb, 1e6, "*10^-6")
show_results("СЕ Довідникові значення", r_se_dv, 1e6, "*10^-6")

=== Розрахункові параметри схеми (СЕ) ===
re = 0.5 Ом
rb = 23.9 Ом
rx = 43477 Ом
μ  = 1.15003 *10^-3
α  = 0.9827
β  = 56.8

=== Розрахункові параметри схеми (СБ) ===
re = -69.6 Ом
rb = 139.4 Ом
rx = 172957 Ом
μ  = 5.87580 *10^-6
α  = 0.4972
β  = 1.0

=== Розрахункові параметри схеми (СЕ Довідникові значення) ===
re = 28.8 Ом
rb = 155.2 Ом
rx = 5680000 Ом
μ  = 387.32591 *10^-6
α  = 0.9859
β  = 70.0



Таблиця вимірів та розрахунків 3.2
| Схема | $r_е,\ Ом$|$r_б,\ Ом$|$r_к,\ Ом$|$µ_{ек}$| $\alpha$ | $\beta$ |
|:--|:-----|:-----|:-----|:-----|:-----|:-----|
| СЕ розрахункове значення | 0.5 | 23.9 | 43473 | $1.15 \cdot 10^{-3}$  | 0.9827 | 56.8 |
| СБ розрахункове значення  | 69.6 | 139.4 | 172957 | $5.875 \cdot 10^{-6}$ | 0.4972 | 1 |
| СЕ довідкове значення  | 28.8 | 155.2 | 5680000 | $387.32 \cdot 10^{-6}$ | 0.985 | 70 |

#### Висновок
У ході лабораторної роботи я дослідив характеристики біполярного транзистора 2N1613 у схемах зі спільним емітером (СЕ) та спільною базою (СБ) за допомогою моделювання. Отримані вхідні та вихідні характеристики підтвердили типову поведінку: у СЕ – високе підсилення струму (β ≈ 57) та помірний вхідний опір, у СБ – низький вхідний опір і коефіцієнт передачі близький до 1. \
Розраховані h-параметри та фізичні параметри еквівалентної схеми загалом узгоджуються з теорією, хоча відрізняються від довідникових значень, ймовірно через особливості моделі в програмі.